src/main/java/uk/org/tombolo/Py4jServer.java

```diff
@@ -0,0 +1,16 @@
+package uk.org.tombolo;
```

Comment:
yeah it does makes sense, however, i thought it would be useful to just put them in respective high level folders. e.g pythonImporter within Importer and rest main files in tombolo with other files as they act as entry points just like DataExportRunner